tests/test_ummg.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, timezone
+
+from freezegun import freeze_time
+
+from mandible.umm.ummg import UmmgBase
+
+
+@freeze_time("2022-08-25 21:45:44.123456")
+def test_ummg():
```

Comment:
Maybe add a test where you make a subclass that returns some of those non-default fields like additional attributes and such.

Reply:
I added an example for a couple on how to use the class. if you think I should add more let me know.